src/index_map.rs

```diff
@@ -90,6 +90,20 @@ impl<K: Eq + Hash, V: StableDeref> FrozenIndexMap<K, V> {
         ret
     }
 
+    pub fn get_index(&self, index: usize) -> Option<(&K::Target, &V::Target)>
+    where
+        V: StableDeref,
```

Comment:
```suggestion
        K: StableDeref,
```

V is already StableDeref on the impl, K is not